wordpress-mobile/WordPress-iOS
# Images copy pasted are duplicated

Fixes #8593 

This PR fixes the duplication of images when copying and pasting. The problem was cause because when an image was detected we where inserting back an attachment instead of updating the already existing attachment.

To test:
 - Create a new post
 - Copy/paste to the post an image from another application (Safari or Notes)
 - Check if the image is added and uploaded to the blog and no duplication show up
